src/main/java/edu/ucla/library/bucketeer/Item.java

```diff
@@ -62,6 +63,9 @@ public Item() {
     @JsonIgnore
     public Item(final String aID, final String aFilePath) {
         myFilePath = Optional.ofNullable(aFilePath);
+        if (myFilePath.isEmpty()) {
+            hasImageFile = false;
```

Comment:
@ksclarke Would it make a difference if we set this member variable to false initially (L41) and set it to true here (`if (myFilePath.isPresent())`) instead?

Reply:
I guess my reason for asking is because a default of `true` there doesn't seem to match with the file path being empty

Reply:
Yes, I think that makes sense.

Reply:
I think that would be good to change for readability, but not essential